orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/AdminController.groovy

```diff
@@ -74,6 +80,29 @@ class AdminController {
     forceExecutionCancellationCommand.forceCancel(executionType, executionId, canceledBy)
   }
 
+  @RequestMapping(value = "/executions", method = RequestMethod.GET)
+  AdminExecutionSearchResult getExecutions(@RequestParam(value = "status", required = true) String status,
+  @RequestParam(value = "startTime", required = true) long startTime,
+  @RequestParam(value = "endTime", required = true) long endTime) {
```

Comment:
This formatting is mega funky.

Reply:
yeah, wtf? just checked that nothing was left behind by spotless.. sigh

Reply:
this is borderline unreadable

Reply:
Groovy does not get formatted, because Groovy is a tire fire.

Reply:
roger that, fixing